packages/design-system/src/components/MonthPicker/MonthPicker.stories.jsx

```diff
@@ -35,7 +35,7 @@ export default {
   },
 };
 
-const Template = ({ data, ...args }) => <MonthPicker {...args} />;
+const Template = (args) => <MonthPicker {...args} />;
```

Comment:
In storybook 6 args are passed first, there are some other places where this could be updated but I was in here messing with passing globals and just left it as it's more correct.